lib/bpk-component-map/src/BpkMapMarkerStyles.js

```diff
@@ -0,0 +1,127 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2020 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import { StyleSheet } from 'react-native';
+import {
+  colorMonteverde,
+  colorSagano,
+  colorGlencoe,
+  borderRadiusSm,
+  borderRadiusPill,
+  borderSizeSm,
+  colorBlackTint03,
+  colorWhite,
+  spacingBase,
+  spacingMd,
+  spacingLg,
+} from 'bpk-tokens/tokens/base.react.native';
+
+import { shadows } from '../../bpk-styles';
+
+const sharedStyles = {
+  wrapper: {
+    ...shadows.base(),
+    paddingHorizontal: spacingLg / 4, // eslint-disable-line backpack/use-tokens
+    paddingVertical: spacingLg / 4, // eslint-disable-line backpack/use-tokens
+  },
+  wrapperFocused: {
+    paddingHorizontal: spacingMd,
+    paddingVertical: spacingMd,
+  },
+  pointer: {
+    backgroundColor: colorMonteverde,
+    borderRadius: borderRadiusSm,
+    height: spacingBase,
+    transform: [{ rotate: '45deg' }],
+    width: spacingBase,
+    left: '38%',
+    position: 'absolute',
+    bottom: -3,
```

Comment:
`38%` and `-3` seem very specific and not tokens. We should at least include a comment to explain why they're set to that, or declare some descriptive constants or something.

Edit: Same for below where `17` is used.

Edit 2: I scrolled down and I see a lot of things like this actually throughout the styles, so same comment for all of those.

Reply:
Sure thing! I tried to make these use tokens or combinations of them but nothing fitted which is annoying and different size things caused it to look either way off design or screws up the alignment/visual glitches on the pointer/wrapper

Reply:
My concern is that with so many 'magic numbers' involved, it'll cause maintenance issues in the future without context as to why these specific values are being used, we won't know how to change the component without breaking it.

Reply:
Oh yeah I am not saying not adding comments I fully agree with that, I was just commenting here for any reviewers right now why 'magic numbers' were used - fully adding comments to the code

Reply:
Ah gotcha!

Reply:
Added some comments and also realised I could make some value changes to use tokens too which helps 😄